### DISCUSSION TITLE
filter restriction

### DISCUSSION THREAD

**Discussion by saba1996** [YetiForceCompany/YetiForceCRM#16903] [General]
How can I limit that the sorting does not change after clicking on the filter?

https://user-images.githubusercontent.com/56167014/219129246-fc0c3beb-758c-4a07-add3-84e17f64f69b.mp4

**Comment by rskrzypczak** [comment]
set the default sorting in the configuration panel for this filter
![image](https://user-images.githubusercontent.com/10221999/219305094-1e2f0d3b-1920-4b25-b8f9-9e27ceebef56.png)

**Reply by saba1996** [reply]
i s

> set the default sorting in the configuration panel for this filter ![image](https://user-images.githubusercontent.com/10221999/219305094-1e2f0d3b-1920-4b25-b8f9-9e27ceebef56.png)

i already do this, but it still changes

**Reply by rskrzypczak** [reply]
try ti log out and log in again :)
